packages/devextreme/js/ui/load_panel.js

```diff
@@ -114,26 +114,33 @@ const LoadPanel = Overlay.inherit({
     },
 
     _render: function() {
+        const { showIndicator } = this.option();
         this.callBase();
 
         this.$element().addClass(LOADPANEL_CLASS);
         this.$wrapper().addClass(LOADPANEL_WRAPPER_CLASS);
 
-        this._setWrapperAria();
+        if(!showIndicator) {
+            this._setWrapperAria();
+        }
     },
 
     _setWrapperAria() {
-        const { message } = this.option();
+        const aria = this._getAriaAttributes();
 
-        const defaultLabel = isMaterialBased() ? message : null;
-        const label = message ? defaultLabel : messageLocalization.format('Loading');
+        this.$wrapper().attr(aria);
+    },
+
+    _getAriaAttributes() {
+        const { message } = this.option();
+        const label = message || messageLocalization.format('Loading');
```

Comment:
This row is not tested at the moment.
Please, add a test checking aria-label when message is set on init and when it's changed in runtime (from empty to not empty and vice versa)

Reply:
[7c89496](https://github.com/DevExpress/DevExtreme/pull/27080/commits/7c89496fe5ea9a3a695c7dcce3717057deeca65c)